nfc/Kconfig

```diff
@@ -4,58 +4,12 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-config NRFXLIB_NFC
-	bool
-	prompt "Enable NFC"
-	select NRFX_NFCT
-	help
-	  Enable NFC functionality
-
-if NRFXLIB_NFC
-
 comment "Common NFC module configuration"
```

Comment:
This does not seem to be needed anymore.

Reply:
Ok, removed.